### PR TITLE
Removes the mess added with recalling with a message

### PR DESCRIPTION
**Pull request by lzimann**

### CODE REVIEW

**Comment by coiax** [comment]
I don't know why you hate msglog_admins so much

**Comment by coiax** [comment]
The codebase is FULL of the two procs with identical contents, differing only in a link or two.

**Comment by Jordie0608** [comment]
It's a lazy 3 lines of snowflake used in one place to save having to write a whole nother line in the rare circumstance have the same text for both `message_admins` and `log_admin`

**Comment by coiax** [comment]
Duplicating strings is bad, it's double the strings to maintain.

**Comment by Cruix** [comment]
Removing duplicate code is the opposite of snowflake and should always be encouraged.

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1193,3 +1193,16 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 	message_admins(msg)
 	admin_ticket_log(target, msg)
 	log_admin("[key_name(usr)] punished [key_name(target)] with [punishment].")
+
+
+/client/proc/trigger_centcom_recall()
+	if(!holder)
```
Comment (Cyberboss) [inline code/modules/admin/verbs/randomverbs.dm]: check_rights(R_ADMIN, TRUE)

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
I agree there' needs to be a common log/msg admin proc but just hamfisting the two together isn't the way to do it

**Comment by optimumtact** [comment]
you broke it good job

**Pull request merged**